src/dmd/root/stringtable.d

```diff
@@ -111,6 +111,11 @@ public:
         return getValue(table[i].vptr);
     }
 
+    inout(StringValue)* lookup(const(char)[] str) inout nothrow pure
```

Comment:
I hate to nitpick, but new public functions should have full DDoc.

Reply:
Done.